.ci/README.md

```diff
@@ -0,0 +1,5 @@
+## Contribution/Development Note
+
+Azure pipeline YAMLs can be validated via a Json schema.
```

Comment:
```suggestion
Azure Pipelines YAML files can be validated via a JSON Schema.
```
Both Azure Pipelines and JSON Schema are names so we should use the official capitalization.

Reply:
Sorry, will change ASAP.